powerhome/playbook
# [NUXE-234] Added React getting started

#### What does this PR do?

1. Adds instructions for React implementation to the Getting Started page.
2. Adds border radius to code block styling on Getting Started page.
3. Adds highlight to Design Component tab for _all_ kit pages.
4. Renames sample pages, fixes imports accordingly.

#### Screens

![react_getting_started](https://p-a6FbDk.t4.n0.cdn.getcloudapp.com/items/6quP4Xww/react_getting_started2.png?v=2015c742c0d26ea6d5f58f4403152c68)

![renamed_samples](https://p-a6FbDk.t4.n0.cdn.getcloudapp.com/items/d5updjWA/renamed_samples.png?v=c38bb72277fbde6bedc3548455797fee)

#### Breaking Changes

n/a

#### Runway Ticket URL

[Link to Runway Story](https://nitro.powerhrg.com/runway/backlog_items/NUXE-234)

#### How to test this

Read over React getting started, double check all kit pages have the Design Components tab highlighted in the full page top nav and the mobile view sidebar nav.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
